spec/integration/parser_spec.rb

```diff
@@ -18,7 +18,39 @@
     end
   end
 
-  def open_file(fixture_file)
-    File.open(File.join(File.dirname(__FILE__), "../#{fixture_file}"))
+  context 'unknown entries' do
+    let(:file_with_unknown_entries) { "fixtures/sie_file_with_unknown_entries.se" }
+
+    context 'lenient parser' do
```

Comment:
Try to keep a consistent quote style, we prefer `"` unless there is a special reason to use `'`.